File: python/example/repl-template.py
# Example REPL Template - The REPL code
# ARBITER set action template
import riffle
from riffle import want

riffle.SetFabricSandbox()

class Backend(riffle.Domain):

    def onJoin(self):
        
        # Exis code goes here
        

app = riffle.Domain("xs.demo.test")

Backend("backend", superdomain=app).join()
# End Example REPL Template

File: js/example/repl-template.js
// Example REPL Template - The REPL code
// ARBITER set action template

var riffle = require('jsriffle');

riffle.SetFabricSandbox();

var app = riffle.Domain("xs.demo.test");
var backend = app.subdomain("backend");

backend.onJoin = function() {

    // Exis code goes here

};

backend.join()
// End Example REPL Template
